src/components/form/index.tsx

```diff
@@ -524,136 +541,6 @@ export const renderField = (props: RenderFieldProps) => {
     );
   }
 
-  if (schema.type === FieldType.unknownProperty) {
```

Comment:
I think we need this unknownProperty support section right? This should be for customData functionality

Reply:
Wierd that this got removed, I don't remember doing that. 🤔

Reply:
I think we just need to put this back and we can probably merge this 🙏

Reply:
Added back in 👍